apps/webapp/README.md

```diff
@@ -0,0 +1,20 @@
+# Starkbot website
+
+Manage your keys from that website.
+
+### Dev setup
+
+Install `npm` and `nodejs` packages according to your distribution.
+create a *.env* file and add the following lines.
+
+```
```

Comment:
give a tool to generate them or give an exemple. Contributors could otherwise generate low-secured keys, which would bring serious issues.